easybuild/easyconfigs/l/LLVM/LLVM-5.0.0-iomkl-2018a.eb

```diff
@@ -0,0 +1,46 @@
+easyblock = 'CMakeMake'
+
+name = 'LLVM'
+version = '5.0.0'
+
+homepage = "http://llvm.org/"
+description = """The LLVM Core libraries provide a modern source- and target-independent
+ optimizer, along with code generation support for many popular CPUs
+ (as well as some less common ones!) These libraries are built around a well
+ specified code representation known as the LLVM intermediate representation
+ ("LLVM IR"). The LLVM Core libraries are well documented, and it is
+ particularly easy to invent your own language (or port an existing compiler)
+ to use LLVM as an optimizer and code generator."""
+
+toolchain = {'name': 'iomkl', 'version': '2018a'}
```

Comment:
I doesn't make much sense to build LLVM with anything else than GCC(core), so you can just remove this easyconfig file (required easyconfig to resolve LLVM 5.0.1 dependency already exists).